app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ActiveRecord::Base
+  before_save { self.email = email.downcase }
```

Comment:
You can move the block into a method and use the method in the callback

For instance, 
before_save :meth

private

def meth
  self.email = email.downcase
end